<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StereoWeidenerFrame</class>
 <widget class="QWidget" name="StereoWeidenerFrame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QToasterDial" name="tuneDial">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="intensityDial">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>10.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy4Dial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Uni</enum>
   </property>
   <property name="minValue">
    <double>0.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>100.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>1.000000000000000</double>
   </property>
   <property name="precision">
    <UInt>0</UInt>
   </property>
   <property name="unit">
    <string/>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy6Dial">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy7Dial">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy8Dial">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="dummy5Dial">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>0</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="knobSize">
    <enum>QToasterDial::Small</enum>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::None</enum>
   </property>
   <property name="isActive">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QToasterDial" name="duckingDial">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>220</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="ledRingType" stdset="0">
    <enum>QToasterDial::Bi</enum>
   </property>
   <property name="minValue">
    <double>-5.000000000000000</double>
   </property>
   <property name="maxValue">
    <double>5.000000000000000</double>
   </property>
   <property name="stepWidth">
    <double>0.100000000000000</double>
   </property>
   <property name="unit">
    <string/>
   </property>
   <property name="isActive">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QToasterStompEditLCD" name="lcdDisplay">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>75</y>
     <width>420</width>
     <height>115</height>
    </rect>
   </property>
   <property name="color">
    <enum>QToasterLCD::Yellow</enum>
   </property>
   <property name="currentPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="maxPage">
    <enum>QToasterLCD::Page1</enum>
   </property>
   <property name="stompName">
    <string/>
   </property>
   <property name="value1Title">
    <string>Intensity</string>
   </property>
   <property name="value2Title">
    <string>Tune</string>
   </property>
   <property name="value3Title">
    <string>Ducking</string>
   </property>
   <property name="value4Title">
    <string/>
   </property>
   <property name="value5Title">
    <string/>
   </property>
   <property name="value6Title">
    <string/>
   </property>
   <property name="value7Title">
    <string/>
   </property>
   <property name="value8Title">
    <string/>
   </property>
   <property name="value9Title">
    <string/>
   </property>
   <property name="value10Title">
    <string/>
   </property>
   <property name="value11Title">
    <string/>
   </property>
   <property name="value12Title">
    <string/>
   </property>
   <property name="value13Title">
    <string/>
   </property>
   <property name="value14Title">
    <string/>
   </property>
   <property name="value15Title">
    <string/>
   </property>
   <property name="value16Title">
    <string/>
   </property>
   <property name="value17Title">
    <string/>
   </property>
   <property name="value18Title">
    <string/>
   </property>
   <property name="value19Title">
    <string/>
   </property>
   <property name="value20Title">
    <string/>
   </property>
   <property name="value21Title">
    <string/>
   </property>
   <property name="value22Title">
    <string/>
   </property>
   <property name="value23Title">
    <string/>
   </property>
   <property name="value24Title">
    <string/>
   </property>
   <property name="scaleView">
    <enum>QToasterStompEditLCD::Empty</enum>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QToasterDial</class>
   <extends>QWidget</extends>
   <header>qtoasterdial.h</header>
  </customwidget>
  <customwidget>
   <class>QToasterStompEditLCD</class>
   <extends>QWidget</extends>
   <header>QToasterStompEditLCD.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>intensityDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue1(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>118</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>111</x>
     <y>148</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tuneDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue2(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>235</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel">
     <x>227</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>duckingDial</sender>
   <signal>valueChanged(QString)</signal>
   <receiver>lcdDisplay</receiver>
   <slot>setValue3(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>270</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>160</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
